data/models/AudioFiles.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * AudioFiles.js
+ *
+ * Data model for storing audio files.
+ * 
+ * Author: Tamara Hills
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+const dynamoose = require('dynamoose');
+
+const schema = new dynamoose.Schema({
+  item_id: {
+    type: String,
+    hashKey: true
+  },
+  uuid: {
+    type: String,
+    rangeKey: true,
+    required: true
+  },
+  type: String, // Can be intro, outro, article, mobile
+  lang: String, // localization identifier for lang (e.g en)
```

Comment:
is 'lang' only relevant for mobile?

Reply:
no, I think it should be available for all platforms.